sceptre/scipool/config/strides/lambda-send-budget-alert.yaml

```diff
@@ -0,0 +1,13 @@
+template_path: remote/lambda-send-budget-alert.yaml
+stack_name: lambda-send-budget-alert
+parameters:
+  SNSTopicARN: !stack_output_external lambda-budgets::BudgetMakerNotificationTopicArn
+  SynapseUserKeyName: '/lambda-send-budget-alert/synapse-username'
+  SynapsePasswordKeyName: '/lambda-send-budget-alert/synapse-password'
```

Comment:
These are parameters in the SSM parameter store.  You'll need to add these to the org-sagebase-strides AWS account.

Reply:
Done!